src/vscode-bicep/src/panes/deploy/models.ts

```diff
@@ -40,3 +43,6 @@ export interface DeployPaneState {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type UntypedError = any;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type TelemetryProperties = Record<string, any>;
```

Comment:
As a best practice, consider replacing `any` with `unknown` and using explicit type casting when necessary.